src/options.ts

```diff
@@ -32,7 +32,7 @@ export type Options = {
   context: boolean;
   snakeToCamel: boolean;
   forceLong: LongOption;
-  useOptionals: boolean;
+  useOptionals: boolean | 'none' | 'messages' | 'all'; // boolean is deprecated
```

Comment:
One option would be to make this type:

```
useOptionals: ['messages' | 'primitives'];
```

And then around in here:

https://github.com/stephenh/ts-proto/blob/main/src/options.ts#L122

Do a translation of `true` -> `[messages, primitives]`, `false` / `none` -> `[]`.

Hm, I guess `parseParameter` is not fancy enough to like detect `,` and do a `.split(`,`)`. It probably could though, and seems like a reusable thing.

If you want to add this as part of this PR, that'd be great, but also fine to merge as-is.